# device or filter
string dev

# the data format:
string type

# name of the package - not required
string package

# name of the executable - not required
string executable

